ci/travis/install-dependencies.sh

```diff
@@ -257,6 +257,12 @@ install_toolchains() {
   fi
 }
 
+download_mnist() {
+  mkdir -p "${HOME}/data"
+  curl -o "${HOME}/data/mnist.zip" https://ray-ci-mnist.s3-us-west-2.amazonaws.com/mnist.zip
+  unzip "${HOME}/data/mnist.zip" -d "${HOME}/data"
```

Comment:
does this look OK? we're hosting this ourselves and the bucket has public read access

Reply:
yeah this is totally fine